File: src/main/java/EKIDS/Lessons/Lesson09/SWING/SimpleWindow.java
package EKIDS.Lessons.Lesson09.SWING;

import java.awt.Dimension;
import java.awt.Toolkit;
import javax.swing.JFrame;

public class SimpleWindow {
    public static void main(String[] args) {
        JFrame simpleWindow = new JFrame();
        simpleWindow.setVisible(true);
        simpleWindow.setTitle("Simple Window");
        simpleWindow.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        Toolkit toolkit = Toolkit.getDefaultToolkit();
        Dimension dimension = toolkit.getScreenSize();
        simpleWindow.setBounds(dimension.width / 2 - 320, dimension.height / 2 - 240, 640, 480);
    }
}

File: src/main/java/EKIDS/Kotlin/Lessons/Lesson10/SWING/SimpleWindow.kt
package Kotlin.Lessons.Lesson10.SWING

import java.awt.Toolkit
import javax.swing.JFrame

fun main() {
    val sW = JFrame()
    sW.isVisible = true
    sW.defaultCloseOperation = JFrame.EXIT_ON_CLOSE
    val toolkit = Toolkit.getDefaultToolkit()
    val dimension = toolkit.screenSize
    sW.setBounds(dimension.width / 2 - 320, dimension.height / 2 - 240,640, 480)
    sW.title = "Simple window"
}